packages/attack-http-requests/README.md

```diff
@@ -0,0 +1,4 @@
+
+# @dazn/chaos-squirrel-attack-http-requests
+
+Attacks HTTP requests.
```

Comment:
Sorry for micro-reviewing - needs some basic usage doc

See one of the other attacks for our bare minimum - https://github.com/getndazn/chaos-squirrel/tree/master/packages/attack-disk-space

Reply:
Good point! 

Added a basic README, would like to do a more broad review over the whole repo. 

![image](https://user-images.githubusercontent.com/5528307/110968405-6419ce80-834f-11eb-945d-98f23bbb1c0f.png)